test/LdapTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
```

Comment:
Should we place this code in OfflineTest or move the contents of OfflineTest here later?